walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -161,30 +161,22 @@
   private void findAndAddGapsInOccupiedTimeRanges(long requestedDuration,
       List<TimePoint> orderedTimePoints,
       Collection<TimeRange> availableTimeRanges) {
-    int index = 1;
-    int unendedTimeRange = 1;
-
-    while (index < orderedTimePoints.size()) {
+    int unendedTimeRange = 0;
+    for (int index = 0; index < orderedTimePoints.size(); index++) {
       // Construct a continuously occupied range of time, which needs to be skipped. Such
       // continuously occupied range of time would contain an equal number of start and end
       // {@code TimePoint}. This occupied range may look like (TR stands for {@code TimeRange}):
       //    [start of TR1, start of TR2, end of TR1, start of TR3, end of TR3, end of TR2]
       // Available time occurs after the continuously occupied ranges of time.
-      if (unendedTimeRange > 0) {
-        if (orderedTimePoints.get(index).isTimeRangeStart()) {
-          unendedTimeRange++;
-        } else {
-          unendedTimeRange--;
-        }
-      } else {
-        int availableTimeRangeStart = orderedTimePoints.get(index - 1).getTime();
-        int availableTimeRangeEnd = orderedTimePoints.get(index).getTime();
+      int _ = orderedTimePoints.get(index).isTimeRangeStart() ?
```

Comment:
Oh does this have to be an assignment? Maybe you can just assign back to unendedTimeRange and do +1 or -1 instead of ++/--

Reply:
Yeah, it must be some type of a statement. Ok! I'll make the change